src/main/java/org/amahi/anywhere/adapter/NavigationDrawerAdapter.java

```diff
@@ -40,14 +57,44 @@ public NavigationDrawerViewHolder onCreateViewHolder(ViewGroup parent, int viewT
         return new NavigationDrawerViewHolder(LayoutInflater.from(parent.getContext()).inflate(R.layout.view_navigation_item, parent, false));
     }
 
+    private Drawable getServerBubble(ArrayList<String> serverList, int position) {
+        //get first letter of each String item
+        String firstLetter = String.valueOf(serverList.get(position).charAt(0));
+
+        ColorGenerator generator = ColorGenerator.MATERIAL; // or use DEFAULT
```

Comment:
why random?

Reply:
I guess random colors are needed for that label thing corresponding to the server name. But there could be a different approach (like creating an array of colors for that), instead of adding a new library.

Reply:
Yep, true. I could have done that with an array of colors for different alphabets. 
And that library does pretty much everything, including making a circle of random color with an appropriate alphabet while being lightweight too.